charmcraft/commands/store/__init__.py

```diff
@@ -240,3 +240,49 @@ def run(self, parsed_args):
         table = tabulate(data, headers=headers, tablefmt='plain', numalign='left')
         for line in table.splitlines():
             logger.info(line)
+
+
+class ReleaseCommand(BaseCommand):
+    """Release a charm revision to specific channels."""
+    name = 'release'
+    help_msg = "relase a charm revision to one or more channels"
+
+    def fill_parser(self, parser):
+        """Add own parameters to the general parser."""
+        parser.add_argument(
+            'channels', metavar='channel', nargs='+',
+            help="the channel(s) to release to")
```

Comment:
The snapcraft release help text at least says it is:
```
      snapcraft release my-snap 9 beta,edge
```

Which is a comma separated list, rather than separat args. I like your way better, but I'm not sure whether we are going by a new design or not.

Reply:
We choose what we liked the best, let's see if UX people have opinions on this...